### ISSUE TITLE
Required fields not working

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#35]
**Description**
Required fields that are empty are saveable.

**Steps to reproduce**
1. Add required field to vizy block (nested or not)
2. Save empty

**Additional info**
- Plugin version: 1.0.2
- Craft version: 3.6.15
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
